internal/core/data/init.go

```diff
@@ -231,12 +232,12 @@ func connectAndPollSecrets() error {
 		return err
 	}
 
-	username, err := secretsClient.GetSecret("coredata")
+	username, err := secretsClient.GetSecret("User")
 	if err != nil {
 		return err
 	}
 
-	password, err := secretsClient.GetSecret("coredatapasswd")
+	password, err := secretsClient.GetSecret("Passwd")
```

Comment:
Should `Passwd` be a constant exposed from the SecretClient? (spell it out please)

Reply:
> Should `Passwd` be a constant exposed from the SecretClient? (spell it out please -- Password)

Good suggestion. @tingyuz, is there something I need to do to update the key that was stored in the Vault instance I was testing the client against? Currently the key is `Passwd` for your test instance.